wandb/integration/keras_core/model_checkpoint.py

```diff
@@ -0,0 +1,175 @@
+import os
+from typing import Dict, List, Literal, Optional, Union
+
+import wandb
+from wandb.sdk.lib import telemetry
+from wandb.sdk.lib.paths import StrPath
+
+try:
+    from keras_core.callbacks import ModelCheckpoint
+except Exception as e:
+    wandb.Error(e)
+
+
+Mode = Literal["auto", "min", "max"]
+SaveStrategy = Literal["epoch"]
+
+
+def _log_artifact(
+    filepath: StrPath,
+    artifact_type: str,
+    aliases: Optional[List] = None,
+    metadata: Optional[Dict] = None,
+) -> None:
+    """Log an artifact to Weights & Biases."""
+    aliases = ["latest"] if aliases is None else aliases + ["latest"]
+    metadata = wandb.run.config.as_dict() if metadata is None else metadata
```

Comment:
since we are passing the metadata as a dict with `logs`. let's update the dict with `wandb.run.config.as_dict`.